### PR TITLE
ragel: new package to build vectorscan

### PR DESCRIPTION
**Pull request by graysky2**
This is a new package for ragel which is a dependency for another new package vectorscan

### CODE REVIEW

**Comment by graysky2** [comment]
Acknowledgment and thanks to @1715173329 for helping me get the syntax correct for this `Makefile`

File: devel/ragel/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2023 OpenWrt.org
```
Comment (1715173329) [inline devel/ragel/Makefile:2]: please either assign the copyright to yourself or simply remove this section.

File: devel/ragel/Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+PKG_NAME:=ragel
+PKG_VERSION:=6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://www.colm.net/files/ragel/
+PKG_SOURCE:=ragel-$(PKG_VERSION).tar.gz
+PKG_HASH:=5f156edb65d20b856d638dd9ee2dfb43285914d9aa2b6ec779dac0270cd56c3f
+
+PKG_MAINTAINER:=John Audia <therealgraysky@proton.me>
+PKG_LICENSE:=GPL
```
Comment (1715173329) [inline devel/ragel/Makefile:19]: from https://www.colm.net/open-source/ragel/ for ragel 6.x the license is `GPL-2.0-only`

Comment (graysky2) [reply devel/ragel/Makefile]: Thanks for that.  I updated the PR.

File: devel/ragel/Makefile (hunk 1 above)
Comment (graysky2) [reply devel/ragel/Makefile]: Removed, thanks for the feedback.

**Review by 1715173329 — APPROVED** [review]

File: devel/ragel/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ragel
+PKG_VERSION:=6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://www.colm.net/files/ragel/
+PKG_SOURCE:=ragel-$(PKG_VERSION).tar.gz
```
Comment (BKPepe) [inline devel/ragel/Makefile:13]:
```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

**Comment by graysky2** [comment]
Would you like me to squash the 3 commits into a single or leave as-is?

**Pull request merged**